docs/src/main/asciidoc/scripting.adoc

```diff
@@ -231,7 +232,7 @@ We are using `curl -w "\n"` in this example to avoid your terminal printing a '%
 ====
 
 [TIP]
-.Why is `quarkus-resteasy` not resolved ? 
+.Why is `quarkus-resteasy` not resolved ?
```

Comment:
```suggestion
.Why is `quarkus-resteasy` not resolved?
```

Reply:
Done, there was in fact another ` ?` I also fixed.